test/domain.test.js

```diff
@@ -61,10 +63,9 @@ test('slds from map file', function (t) {
 })
 
 test('punycode', function (t) {
-  t.plan(15)
+  t.plan(14)
   t.equal(isValidDomain('xn--6qq79v.xn--fiqz9s'), true)
   t.equal(isValidDomain('xn--ber-goa.com'), true)
-  t.equal(isValidDomain('xn--a--ber-goa.com'), false)
```

Comment:
Sigh... I read this too quickly and thought it was a subdomain. This fix probably isn't done until this passes.

Reply:
Oh - I deleted this test erroneously when trying to fix it, and the end solution actually passes this test. Adding it back.